llvm/test/tools/llvm-dwarfdump/X86/DW_AT_language_version.s

```diff
@@ -1,13 +1,18 @@
 # Demonstrate dumping DW_AT_language_version.
```

Comment:
This is the case where we don't recognize the language or version code? perhaps the test file name and/or header comment could clarify that this is about dumping unrecognized language/version codes?